src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -9,17 +9,40 @@
 
 public class Ladder {
     private final List<Line> lines;
+    private final Members members;
 
-    public Ladder(int countOfNumber, int height) {
-        lines = IntStream.range(0, height)
-                         .mapToObj(h -> new Line(countOfNumber, new DefaultRandomBoolean()))
-                         .collect(Collectors.toList());
+    public Ladder(Members members, int height) {
+        this.members = members;
+        this.lines = IntStream.range(0, height)
+                              .mapToObj(h -> new Line(members.countOfMember(), new DefaultRandomBoolean()))
+                              .collect(Collectors.toList());
```

Comment:
```suggestion
        this.lines = Stream.generate(() -> new Line(members.countOfMember(), new DefaultRandomBoolean()))
                .limit(height)
                .collect(Collectors.toList());
```
이런 방식도 가능해요, 참고만 해주세요~!

Reply:
오 이런것도 있군요 감사합니다 :)